Android/app/src/go/logging/logging.go

```diff
@@ -12,27 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package doh
+package logging
 
 import (
-	"sync/atomic"
+	"io"
+	"log"
+	"os"
 )
 
-// Atomic is atomic.Value, specialized for doh.Transport.
-type Atomic struct {
-	v atomic.Value
-}
-
-// Store a DNSTransport.  d must not be nil.
-func (a *Atomic) Store(t Transport) {
-	a.v.Store(t)
-}
-
-// Load the DNSTransport, or nil if it has not been stored.
-func (a *Atomic) Load() Transport {
-	v := a.v.Load()
-	if v == nil {
-		return nil
-	}
-	return v.(Transport)
-}
+var Debug = log.New(io.Discard, "[DEBUG] ", log.LstdFlags)
```

Comment:
This is not an efficient way to define the loggers, because you will still create the messages.

We need leveled logging instead.

Consider using slog, which is more efficient:
https://pkg.go.dev/log/slog
https://go.dev/blog/slog